src/java/fr/paris/lutece/plugins/forms/validation/AbstractNumberValidator.java

```diff
@@ -0,0 +1,78 @@
+package fr.paris.lutece.plugins.forms.validation;
+
+import fr.paris.lutece.plugins.forms.business.Control;
+import fr.paris.lutece.plugins.forms.business.FormQuestionResponse;
+import fr.paris.lutece.plugins.forms.util.FormsConstants;
+import fr.paris.lutece.plugins.genericattributes.business.Response;
+import fr.paris.lutece.portal.service.i18n.I18nService;
+import fr.paris.lutece.portal.service.template.AppTemplateService;
+import fr.paris.lutece.portal.service.util.AppLogService;
+import fr.paris.lutece.util.html.HtmlTemplate;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public abstract class AbstractNumberValidator extends AbstractValidator
+{
+    private static final String TEMPLATE_DISPLAY_HTML = "/admin/plugins/forms/validators/number_value_template.html";
+
+    private static final String MARK_LABEL = "label_key";
+
+    private static final String MARK_HELP = "help_key";
+
+    /**
+     * Constructor of the AbstractValidator
+     *
+     * @param strValidatorName
+     *         The validator bean name
+     * @param strValidatorDisplayName
+     *         The validator display name
+     * @param listAvailableEntryType
+     *         The list of available entrytype
+     */
+    protected AbstractNumberValidator( String strValidatorName, String strValidatorDisplayName, List<String> listAvailableEntryType )
+    {
+        super( strValidatorName, strValidatorDisplayName, listAvailableEntryType );
+    }
+
+    @Override
+    public String getDisplayHtml( Control control )
+    {
+        Map<String, Object> model = new HashMap<>( );
+        model.put( FormsConstants.PARAMETER_CONTROL_VALUE, control.getValue( ) );
+        model.put( MARK_LABEL, getLabelKey( ) );
+        model.put( MARK_HELP, getHelpKey( ) );
+
+        HtmlTemplate htmlTemplateQuestion = AppTemplateService.getTemplate( TEMPLATE_DISPLAY_HTML, I18nService.getDefaultLocale( ), model );
```

Comment:
Current user may have chosen another locale;
Add a locale parameter in a second constructor of AbstractValidator,
Make the first constructor call this second constructor with this default.